packages/components/FocusZone/src/FocusZone.ts

```diff
@@ -15,7 +15,7 @@ const filterOutComponentRef = (propName) => propName !== 'componentRef';
 
 export const FocusZone = composable<FocusZoneType>({
   usePrepareProps: (userProps: FocusZoneProps, useStyling: IUseStyling<FocusZoneType>) => {
-    const { componentRef, defaultTabbableElement, isCircularNavigation, ...rest } = userProps;
+    const { componentRef, defaultTabbableElement, isCircularNavigation, tabKeyNavigation, ...rest } = userProps;
```

Comment:
looks like it could be left in `...rest`?

Reply:
You're correct - changing in newest iteration